### PR TITLE
[zh]sync contribution files for 1.22(Part-1)

### PR DESCRIPTION
**Pull request by howieyuen**
related https://github.com/kubernetes/website/issues/29326

### CODE REVIEW

File: content/zh/docs/contribute/new-content/open-a-pr.md [hunk 1]
```diff
@@ -236,19 +236,19 @@ Make sure you have [git](https://git-scm.com/book/en/v2/Getting-Started-Installi
    这样可以确保你本地的仓库在开始工作前是最新的。
 
    <!--
-    This workflow is different than the [Kubernetes Community GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md). You do not need to merge your local copy of `master` with `upstream/master` before pushing updates to your fork.
+    This workflow is different than the [Kubernetes Community GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md). You do not need to merge your local copy of `main` with `upstream/main` before pushing updates to your fork.
    -->
     {{< note >}}
-    此工作流程与 [Kubernetes 社区 GitHub 工作流](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)有所不同。在推送你的变更到你的远程派生副本库之前，你不需要将你本地的 `master` 与 `upstream/master` 合并。
+    此工作流程与 [Kubernetes 社区 GitHub 工作流](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)有所不同。在推送你的变更到你的远程派生副本库之前，你不需要将你本地的 `main` 与 `upstream/main` 合并。
```
Comment (tengqm) [inline content/zh/docs/contribute/new-content/open-a-pr.md:242]: Wrapping the long line would be a plus

**Review by tengqm — COMMENTED** [review]
/approve

File: content/zh/docs/contribute/new-content/open-a-pr.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/contribute/new-content/open-a-pr.md]: fixed

**Pull request merged**